cms/blocks.py

```diff
@@ -46,6 +46,23 @@ class UserTestimonialBlock(blocks.StructBlock):
     quote = blocks.TextBlock(help_text="The quote that appears on the testimonial.")
 
 
+class NewsAndEventsBlock(blocks.StructBlock):
+    """
+    Custom block to represent a news or event
+    """
+
+    content_type = blocks.CharBlock(max_length=100, help_text="The type of update.")
```

Comment:
suggestion: I would prefer `Specify the news/events type here`

Reply:
updated.